admin.php

```diff
@@ -41,218 +41,191 @@ function sqlQuery($statement, $link)
-
-                    echo " </tr>\n";
-
-                    if ($config && $dbh !== false) {
-                        mysqli_close($dbh);
-                    }
-                }
-                ?>
-                </table>
+                        ?>
+                    </table>
+                </div>
                 <form method='post' action='setup.php'>
-                     <button type='submit' name='form_submit' value='Add New Site'><b>Add New Site</b></button>
+                    <button type='submit' class='btn btn-primary font-weight-bold' name='form_submit' value='Add New Site'>Add New Site</button>
                 </form>
             </div>
         </div>
     </div><!--end of container div-->
+
     <div class="row">
-            <div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
-                <div class="modal-dialog modal-lg">
-                    <div class="modal-content  oe-modal-content" style="height:700px">
-                        <div class="modal-header clearfix">
-                            <button type="button" class="close" data-dismiss="modal" aria-label=Close>
-                            <span aria-hidden="true" style="color:var(--black); font-size:1.5em;">×</span></button>
-                        </div>
+        <div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+            <div class="modal-dialog modal-lg">
+                <div class="modal-content" style="height:700px">
+                    <div class="modal-header clearfix">
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                            <span aria-hidden="true" style="color:var(--black); font-size:1.5em;">×</span>
+                        </button>
+                    </div>
                         <div class="modal-body" style="height:80%;">
-                            <iframe src="" id="targetiframe" style="height:100%; width:100%; overflow-x: hidden; border:none"
-                            allowtransparency="true"></iframe>
-                        </div>
-                        <div class="modal-footer" style="margin-top:0px;">
-                           <button class="btn btn-link btn-cancel oe-pull-away" data-dismiss="modal" type="button">Close</button>
-                           <!--<button class="btn btn-secondary btn-print oe-pull-away" data-dismiss="modal" id="print-help-href" type="button">Print</button>-->
+                            <iframe src="" id="targetiframe" class="h-100 w-100" style="overflow-x: hidden; border:none"
+                                allowtransparency="true"></iframe>
                         </div>
+                    <div class="modal-footer mt-0">
+                        <button class="btn btn-secondary btn-cancel" data-dismiss="modal" type="button">Close</button>
                     </div>
```

Comment:
Also, I found the `btn-cancel` our custome font-awesome class didn't work on this page, so should I remove it?

Reply:
yep, would remove there since this script is runs independent from OpenEMR context and doesn't have access (since it is not able to bring in globals.php).

Reply:
OK, I removed it and updated the commit  :)